Resources/Private/Partials/Form/ValidationErrors.html

```diff
@@ -0,0 +1,12 @@
+<div class="alert alert-danger">
+  <p>
+    <f:translate key="resetPassword.errors">There where some errors with your input:</f:translate>
```

Comment:
Please use a generic translation key here. This partial will be used in other locations, too.